ipaserver/install/ipa_migrate_constants.py

```diff
@@ -711,6 +726,13 @@
         'subtree': ',cn=computers,cn=accounts,$SUFFIX',
         'label': 'Hosts',
         'mode': 'all',
+        'preserve_attrs': [
+            'cn',
+            'fqdn',
+            'dn',
+            'krbprincipalname',
+            'krbcanonicalname',
```

Comment:
A principal contains the FQDN plus the realm. Will the realm still be replaced with whatever the new one is?

Reply:
It should be possible to replace just the realm and not the hostname as long as the realm is _always_ CAPITALIZED